app/src/main/java/org/mozilla/fenix/components/toolbar/TabCounterToolbarButton.kt

```diff
@@ -31,14 +31,19 @@ class TabCounterToolbarButton(
             setOnClickListener {
                 showTabs.invoke()
             }
-            contentDescription =
-                parent.context.getString(R.string.mozac_feature_tabs_toolbar_tabs_button)
+
+            val count = sessionManager.sessions.count {
+                it.private == isPrivate
+            }
+
+            contentDescription = if (count > 1) context?.getString(
```

Comment:
Could getting the string be moved to a helper since its done in 2 different places?